### DISCUSSION TITLE
Question 7: Taxonomy Module - Two authors and two years? [RC]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#748] [Q&A] [answered]
Description: I was working on the Taxonomy module and I found the following information on Worms: Arbacia stellata (Blainville, 1825; ?Gmelin, 1791)

- How do I add this to the database? Two authors and two years?

**Comment by rondlg** [comment, marked as answer]
Response: [JJ] will email with an answer, but if this is an issue with a questionable authorship it might be appropriate to use Notes to record this until a decision is reached.

- Semi-related?: https://github.com/fieldmuseum/EMu-Documentation/discussions/660

**Reply by fmjjones** [reply]
**If** Arbacia stellata is the accepted name of [Echinus stellata](https://www.marinespecies.org/aphia.php?p=taxdetails&id=569013) ([original description]( http://www.biodiversitylibrary.org/item/83098#5)) then the author and year are (Gmelin, 1791), however there seems to be some question in reference Taxonomic citation
Kroh, A.; Mooi, R. (2023). World Echinoidea Database. Arbacia stellata (Blainville, 1825; ?Gmelin, 1791). Accessed through: World Register of Marine Species at: https://www.marinespecies.org/aphia.php?p=taxdetails&id=513125 on 2023-01-10.

I would not include an author and year in the taxonomic record until that issue is either resolved or explained. Add the author/Year information to the notes field with the "Kind" = Authority.
